### PR TITLE
ACN_CD_SymantecDLP_Connector01

### PR DESCRIPTION
**Pull request by chicduong**
## Proposed Changes

  - Symantec DLP Data Connector

### CODE REVIEW

**Comment by preetikr** [comment]
@chicduong - Please share sample data for this data connector.

**Comment by chicduong** [comment]
@preetikr I submitted the sample data with the parser. From a previous Parser PR @shainw had indicated he was missing Sample data. Which PR should I include the sample data with, Connector or Parser?

**Comment by preetikr** [comment]
@chicduong - It's fine to submit with the parser PR - please add a link to the PR that has the sample data so that the reviewer of this data connector is aware that sample data is available as well. Thanks.

**Comment by chicduong** [comment]
sample data included in PR https://github.com/Azure/Azure-Sentinel/pull/1364

File: DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+    "id": "SymantecDLP",
+    "title": "Symantec DLP",
+    "publisher": "Broadcom",
```
Comment (v-jayakal) [inline DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json:4]: Publisher name should be part of id and title. Please refer https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Data%20Connectors%20Template%20Guidance.md for guidelines.

File: DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "sampleQueries": [{
+        "description": "Top 10 Triggered Activities",
+        "query": "SymantecDLP \n | summarize count() by Activity \n| top 10 by count_"
+    }, {
+        "description": "Top 10 Filenames",
+        "query": "SymantecDLP \n | summarize count() by FileName \n| top 10 by count_"
+    }],
+    "connectivityCriterias": [{
+        "type": "IsConnectedQuery",
+        "value": ["CommonSecurityLog​ \n| where DeviceVendor == \"Symantec\" and DeviceProduct == \"DLP\"\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"]
```
Comment (v-jayakal) [inline DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json:21]: Please remove [ before CommonSecurityLog and ] at the end, they are not required.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
+{
+    "id": "SymantecDLP",
+    "title": "Symantec DLP",
+    "publisher": "Broadcom",
+    "descriptionMarkdown": "The [Broadcom Symantec Data Loss Prevention (DLP)](https://www.broadcom.com/products/cyber-security/information-protection/data-loss-prevention) connector allows you to easily connect your Symantec DLP with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization’s information, where it travels, and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **SymantecDLP** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-symantecdlp-parser)",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "Symantec DLP",
```
Comment (v-jayakal) [inline DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json:9]: Legend value should be SymantecDLP (remove space) as it is DATATYPE_NAME. Refer https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
+{
+    "id": "SymantecDLP",
```
[the remark this answers is not in the document]
Comment (chicduong) [reply DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json:2]: changes have been pushed

File: DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json (hunk 1 above)
Comment (chicduong) [reply DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json:4]: changes have been pushed

File: DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json (hunk 2 above)
Comment (chicduong) [reply DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json]: changes have been pushed. This is taken from the template from the Palo Alto connector page.

File: DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json (hunk 3 above)
Comment (chicduong) [reply DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json]: changes have been pushed

**Review by v-jayakal — APPROVED** [review]

**Review by nazang — APPROVED** [review]

**Pull request merged**